QLSettings.h

```diff
@@ -69,6 +71,7 @@ class QLSettings {
 	int32	fSingleClick;
 	int32	fOnTop;
 	int32	fShowIgnore;
+	int32	fSearchStart;
 
 	BLocker	fLock;
 };
```

Comment:
All these functions and variables should be ordered to reflect the order in the GUI, too...